### PR TITLE
Bump postcss-values-replace

### PR DESCRIPTION
**Pull request by SkReD**
Closes https://github.com/cssnano/cssnano/issues/909

### CODE REVIEW

**Review by evilebottnawi — APPROVED** [review]
Thanks

/cc @anikethsaha

**Review by evilebottnawi — APPROVED** [review]

File: packages/postcss-convert-values/src/index.js [hunk 1]
```diff
@@ -24,7 +24,12 @@ function parseWord(node, opts, keepZeroUnit) {
   const pair = unit(node.value);
   if (pair) {
     const num = Number(pair.number);
-    const u = pair.unit;
+    let u = pair.unit;
+
+    if (u[0] === '.') {
+      u = u.slice(1);
+    }
+
```
Comment (anikethsaha) [inline packages/postcss-convert-values/src/index.js:32]: Why removing the leading `.` ?

**Review by anikethsaha — CHANGES_REQUESTED** [review]
One question. 
And also fix the linting issue

File: packages/postcss-convert-values/src/index.js (hunk 1 above)
Comment (SkReD) [reply packages/postcss-convert-values/src/index.js:32]: my thought  is that is not the part of `unit` but the `value`

Comment (anikethsaha) [reply packages/postcss-convert-values/src/index.js:32]: for input like https://github.com/TrySound/postcss-value-parser/blob/master/test/unit.js#L10, there would be still number (length) in the unit. 

I wonder whether this should be fixed in `postcss-value-parser` as `.3rem` or `3rem` is even a value unit or not. 

cc @evilebottnawi

**Review by anikethsaha — CHANGES_REQUESTED** [review]

File: packages/postcss-convert-values/src/index.js (hunk 1 above)
Comment (anikethsaha) [reply packages/postcss-convert-values/src/index.js:32]: @evilebottnawi

**Review by anikethsaha — CHANGES_REQUESTED** [review]
Could you please add a test case mentioned here https://github.com/cssnano/cssnano/issues/958

File: packages/postcss-convert-values/src/index.js (hunk 1 above)
Comment (evilebottnawi) [reply packages/postcss-convert-values/src/index.js:32]: Yes, it is not good idea

Comment (anikethsaha) [reply packages/postcss-convert-values/src/index.js:32]: @SkReD can you undo this change and we can submit a issue in the parser repo as this should be handled by the parser not by the minifier

**Comment by ludofischer** [comment]
`css-value-parser` has been updated on master already but I think the test cases in this issue are interesting. Should check what happens with the current rc.

**Comment by ludofischer** [comment]
Fixed in 5.0.0. I've added the commit with the test case to https://github.com/cssnano/cssnano/pull/1059

**Pull request closed without merging**